### PR TITLE
W-12668701-Update-alert-limit-explanation-CP

### PR DESCRIPTION
**Pull request by Crispy-Salesforce**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Comment by Crispy-Salesforce** [comment]
Parvez approved these changes: https://salesforce-internal.slack.com/archives/C064DSREJCT/p1699980724723909

File: gateway/1.5/modules/ROOT/pages/flex-use-api-alerts.adoc [hunk 1]
```diff
@@ -62,13 +62,21 @@ To view and delete alerts, ensure that you have:
 
 To track and manage APIs, you can add one or more API alerts to your API instances. Add different <<types, types of alerts>> based on your requirements.
 
-Users can have six to ten active alerts. Pausing or deleting an alert allows you to create new alerts once you have reached the maximum number of alerts.
+In the following table, you can learn the amount and type of alerts per API instance.
```
Comment (chris-campbell9) [inline gateway/1.5/modules/ROOT/pages/flex-use-api-alerts.adoc]:
```suggestion
The following table explains the amount and types of alerts per API instance:
```

File: gateway/1.5/modules/ROOT/pages/flex-use-api-alerts.adoc [hunk 2]
```diff
@@ -62,13 +62,21 @@ To view and delete alerts, ensure that you have:
...
+| User | Non-Active Alerts | Active Alerts
+| Gold | No limit | Up to 6
+| Platinum | No limit | Up to 6
+| Titanium | No limit | Up to 10
+|===
+
+Pausing or deleting an active alert enables you to create new alerts if you have reached the maximum number of active alerts. 
 
-Alerts cannot be added to API instances protected by xref:api-manager::automated-policies-landing-page.adoc[automated policies].
+Alerts are enabled by default when they are created and this setting cannot be modified.
```
Comment (chris-campbell9) [inline gateway/1.5/modules/ROOT/pages/flex-use-api-alerts.adoc]:
```suggestion
Alerts are enabled by default when created. This setting cannot be modified.
```

File: gateway/1.5/modules/ROOT/pages/flex-use-api-alerts.adoc [hunk 3]
```diff
@@ -62,13 +62,21 @@ To view and delete alerts, ensure that you have:
 
 To track and manage APIs, you can add one or more API alerts to your API instances. Add different <<types, types of alerts>> based on your requirements.
 
-Users can have six to ten active alerts. Pausing or deleting an alert allows you to create new alerts once you have reached the maximum number of alerts.
+In the following table, you can learn the amount and type of alerts per API instance.
 
-The maximum number of alerts you can create per API instance depends on your plan. Organizations that have a Gold or Platinum subscription can have up to six active alerts per API instance. Organizations that have a Titanium subscription can have up to ten active alerts per API instance. With all plans, deleting or pausing alerts allows you to create new ones.
+[%header%autowidth.spread]
+|===
+| User | Non-Active Alerts | Active Alerts
```
Comment (glenn-rodgers-sf) [inline gateway/1.5/modules/ROOT/pages/flex-use-api-alerts.adoc]:
```suggestion
|Subscription | Non-Active Alerts | Active Alerts
```

File: gateway/1.5/modules/ROOT/pages/flex-use-api-alerts.adoc (hunk 1 above)
Comment (Crispy-Salesforce) [reply gateway/1.5/modules/ROOT/pages/flex-use-api-alerts.adoc]: OK! Thanks!

File: gateway/1.5/modules/ROOT/pages/flex-use-api-alerts.adoc (hunk 3 above)
Comment (Crispy-Salesforce) [reply gateway/1.5/modules/ROOT/pages/flex-use-api-alerts.adoc]: OK! Thanks!

File: gateway/1.5/modules/ROOT/pages/flex-use-api-alerts.adoc (hunk 2 above)
Comment (Crispy-Salesforce) [reply gateway/1.5/modules/ROOT/pages/flex-use-api-alerts.adoc]: OK! Thanks!

**Review by JennyHajee — APPROVED** [review]

**Pull request merged**